asreview/webapp/start_flask.py

```diff
@@ -102,16 +114,62 @@ def create_app(**kwargs):
 
     # Get the ASReview arguments.
     app.config['asr_kwargs'] = kwargs
+    app.config['LOGIN_DISABLED'] = not kwargs['enable_auth']
+
+    # config JSON Web Tokens
+    login_manager = LoginManager(app)
+    login_manager.init_app(app)
+    login_manager.session_protection = 'strong'
+
+    # setup all database/authentication related resources
+    if app.config['LOGIN_DISABLED'] == False:
+
+        # default config
+        app.config['SESSION_COOKIE_SAMESITE'] = 'Lax'
+        app.config['SESSION_COOKIE_SECURE'] = True
+        app.config['REMEMBER_COOKIE_SECURE'] = True
+        app.config['SECRET_KEY'] = os.environ.get(
+            'SECRET_KEY',
+            'JeMoederHeetHenk1!'
+        )
+
+        # setup the database
+        uri = os.path.join(asreview_path(), f'asreview.{env}.sqlite')
+        app.config['SQLALCHEMY_DATABASE_URI'] = f'sqlite:///{uri}'
+        app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = True
+
+        # create the database plus table(s)
+        DB.init_app(app)
+        with app.app_context():
+            DB.create_all()
+    else:
+        # Don't want to use the standard Anonymous User
+        login_manager.anonymous_user = SingleUser
+
+
+    # Register a callback function for current_user.
+    @login_manager.user_loader
+    def load_user(user_id):
+        return User.query.get(int(user_id))
```

Comment:
In the documentation they say this should not raise an error, but return None:
https://flask-login.readthedocs.io/en/latest/#how-it-works
If you call this with something other than an integer it will raise a ValueError. Does this matter?